<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Notification</class>
 <widget class="QDialog" name="Notification">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>297</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowOpacity">
   <double>0.900000000000000</double>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="lay_titlebar" stretch="0">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,100,40">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetMaximumSize</enum>
       </property>
       <item>
        <widget class="QLabel" name="lbl_logo">
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout" rowstretch="0,1">
         <property name="margin">
          <number>6</number>
         </property>
         <property name="horizontalSpacing">
          <number>6</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="lbl_topic">
           <property name="text">
            <string>TextLabel</string>
           </property>
           <property name="margin">
            <number>5</number>
           </property>
           <property name="openExternalLinks">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="lbl_message">
           <property name="text">
            <string>&lt;style  type=&quot;text/css&quot; &gt;
&lt;/style&gt;  
&lt;table id=&quot;T_6d631314_5a1b_11e8_83fe_7824af8f92da&quot; &gt; 
&lt;thead&gt;    &lt;tr&gt; 
        &lt;th class=&quot;blank level0&quot; &gt;&lt;/th&gt; 
        &lt;th class=&quot;col_heading level0 col0&quot; &gt;5&lt;/th&gt; 
    &lt;/tr&gt;&lt;/thead&gt; 
&lt;tbody&gt;    &lt;tr&gt; 
        &lt;th id=&quot;T_6d631314_5a1b_11e8_83fe_7824af8f92dalevel0_row0&quot; class=&quot;row_heading level0 row0&quot; &gt;Номер авто&lt;/th&gt; 
        &lt;td id=&quot;T_6d631314_5a1b_11e8_83fe_7824af8f92darow0_col0&quot; class=&quot;data row0 col0&quot; &gt;А009УР197&lt;/td&gt; 
    &lt;/tr&gt;    &lt;tr&gt; 
        &lt;th id=&quot;T_6d631314_5a1b_11e8_83fe_7824af8f92dalevel0_row1&quot; class=&quot;row_heading level0 row1&quot; &gt;Заказчик&lt;/th&gt; 
        &lt;td id=&quot;T_6d631314_5a1b_11e8_83fe_7824af8f92darow1_col0&quot; class=&quot;data row1 col0&quot; &gt;Симонян&lt;/td&gt; 
    &lt;/tr&gt;    &lt;tr&gt; 
        &lt;th id=&quot;T_6d631314_5a1b_11e8_83fe_7824af8f92dalevel0_row2&quot; class=&quot;row_heading level0 row2&quot; &gt;Дата по&lt;/th&gt; 
        &lt;td id=&quot;T_6d631314_5a1b_11e8_83fe_7824af8f92darow2_col0&quot; class=&quot;data row2 col0&quot; &gt;1970-01-03&lt;/td&gt; 
    &lt;/tr&gt;    &lt;tr&gt; 
        &lt;th id=&quot;T_6d631314_5a1b_11e8_83fe_7824af8f92dalevel0_row3&quot; class=&quot;row_heading level0 row3&quot; &gt;Дата от&lt;/th&gt; 
        &lt;td id=&quot;T_6d631314_5a1b_11e8_83fe_7824af8f92darow3_col0&quot; class=&quot;data row3 col0&quot; &gt;1970-01-01&lt;/td&gt; 
    &lt;/tr&gt;    &lt;tr&gt; 
        &lt;th id=&quot;T_6d631314_5a1b_11e8_83fe_7824af8f92dalevel0_row4&quot; class=&quot;row_heading level0 row4&quot; &gt;Описание&lt;/th&gt; 
        &lt;td id=&quot;T_6d631314_5a1b_11e8_83fe_7824af8f92darow4_col0&quot; class=&quot;data row4 col0&quot; &gt;None&lt;/td&gt; 
    &lt;/tr&gt;    &lt;tr&gt; 
        &lt;th id=&quot;T_6d631314_5a1b_11e8_83fe_7824af8f92dalevel0_row5&quot; class=&quot;row_heading level0 row5&quot; &gt;Эко класс&lt;/th&gt; 
        &lt;td id=&quot;T_6d631314_5a1b_11e8_83fe_7824af8f92darow5_col0&quot; class=&quot;data row5 col0&quot; &gt;None&lt;/td&gt; 
    &lt;/tr&gt;    &lt;tr&gt; 
        &lt;th id=&quot;T_6d631314_5a1b_11e8_83fe_7824af8f92dalevel0_row6&quot; class=&quot;row_heading level0 row6&quot; &gt;Скрыто&lt;/th&gt; 
        &lt;td id=&quot;T_6d631314_5a1b_11e8_83fe_7824af8f92darow6_col0&quot; class=&quot;data row6 col0&quot; &gt;False&lt;/td&gt; 
    &lt;/tr&gt;    &lt;tr&gt; 
        &lt;th id=&quot;T_6d631314_5a1b_11e8_83fe_7824af8f92dalevel0_row7&quot; class=&quot;row_heading level0 row7&quot; &gt;Владелец&lt;/th&gt; 
        &lt;td id=&quot;T_6d631314_5a1b_11e8_83fe_7824af8f92darow7_col0&quot; class=&quot;data row7 col0&quot; &gt;None&lt;/td&gt; 
    &lt;/tr&gt;    &lt;tr&gt; 
        &lt;th id=&quot;T_6d631314_5a1b_11e8_83fe_7824af8f92dalevel0_row8&quot; class=&quot;row_heading level0 row8&quot; &gt;Оплата&lt;/th&gt; 
        &lt;td id=&quot;T_6d631314_5a1b_11e8_83fe_7824af8f92darow8_col0&quot; class=&quot;data row8 col0&quot; &gt;nan&lt;/td&gt; 
    &lt;/tr&gt;    &lt;tr&gt; 
        &lt;th id=&quot;T_6d631314_5a1b_11e8_83fe_7824af8f92dalevel0_row9&quot; class=&quot;row_heading level0 row9&quot; &gt;Цена&lt;/th&gt; 
        &lt;td id=&quot;T_6d631314_5a1b_11e8_83fe_7824af8f92darow9_col0&quot; class=&quot;data row9 col0&quot; &gt;nan&lt;/td&gt; 
    &lt;/tr&gt;    &lt;tr&gt; 
        &lt;th id=&quot;T_6d631314_5a1b_11e8_83fe_7824af8f92dalevel0_row10&quot; class=&quot;row_heading level0 row10&quot; &gt;Заглушенно&lt;/th&gt; 
        &lt;td id=&quot;T_6d631314_5a1b_11e8_83fe_7824af8f92darow10_col0&quot; class=&quot;data row10 col0&quot; &gt;False&lt;/td&gt; 
    &lt;/tr&gt;    &lt;tr&gt; 
        &lt;th id=&quot;T_6d631314_5a1b_11e8_83fe_7824af8f92dalevel0_row11&quot; class=&quot;row_heading level0 row11&quot; &gt;Статус&lt;/th&gt; 
        &lt;td id=&quot;T_6d631314_5a1b_11e8_83fe_7824af8f92darow11_col0&quot; class=&quot;data row11 col0&quot; &gt;None&lt;/td&gt; 
    &lt;/tr&gt;    &lt;tr&gt; 
        &lt;th id=&quot;T_6d631314_5a1b_11e8_83fe_7824af8f92dalevel0_row12&quot; class=&quot;row_heading level0 row12&quot; &gt;Номер СТС&lt;/th&gt; 
        &lt;td id=&quot;T_6d631314_5a1b_11e8_83fe_7824af8f92darow12_col0&quot; class=&quot;data row12 col0&quot; &gt;77 ХС 842429&lt;/td&gt; 
    &lt;/tr&gt;    &lt;tr&gt; 
        &lt;th id=&quot;T_6d631314_5a1b_11e8_83fe_7824af8f92dalevel0_row13&quot; class=&quot;row_heading level0 row13&quot; &gt;Путь&lt;/th&gt; 
        &lt;td id=&quot;T_6d631314_5a1b_11e8_83fe_7824af8f92darow13_col0&quot; class=&quot;data row13 col0&quot; &gt;None&lt;/td&gt; 
    &lt;/tr&gt;    &lt;tr&gt; 
        &lt;th id=&quot;T_6d631314_5a1b_11e8_83fe_7824af8f92dalevel0_row14&quot; class=&quot;row_heading level0 row14&quot; &gt;Зона&lt;/th&gt; 
        &lt;td id=&quot;T_6d631314_5a1b_11e8_83fe_7824af8f92darow14_col0&quot; class=&quot;data row14 col0&quot; &gt;СК&lt;/td&gt; 
    &lt;/tr&gt;&lt;/tbody&gt; 
&lt;/table&gt; </string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
           </property>
           <property name="margin">
            <number>5</number>
           </property>
           <property name="openExternalLinks">
            <bool>true</bool>
           </property>
           <property name="textInteractionFlags">
            <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse|Qt::TextBrowserInteraction|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="btn_close">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>75</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Close</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
